client/src/scenes/Question/scenes/Read/Read.jsx

```diff
@@ -32,7 +35,13 @@ const Read = ({ history, match, zNode, loading }) => {
   }, [loaded, incremented, incrementViewsCounter, zNode])
 
   useEffect(() => {
-    if (!loaded && zNode) setLoaded(true)
+    if (!loaded && zNode) {
+      setLoaded(true)
+      setQuestionTitle(zNode.question.title)
+      {
+        zNode.answer && setAnswerContent(zNode.answer.content)
+      }
```

Comment:
Je pense que ce serait plus clair comme ça

```suggestion
      if (zNode.answer) {
        setAnswerContent(zNode.answer.content)
      }
```